haystack/nodes/prompt/providers.py

```diff
@@ -76,10 +87,6 @@ def _ensure_token_limit(self, prompt: str) -> str:
         pass
 
 
-def known_providers() -> List[Type[PromptModelInvocationLayer]]:
-    return [HFLocalInvocationLayer, OpenAIInvocationLayer, AzureOpenAIInvocationLayer]
-
-
```

Comment:
No need for this one as well because these three are going to always be registered in `PromptModelInvocationLayer.invocation_layer_providers`